feature-libs/product-configurator/rulebased/components/overview-filter/configurator-overview-filter.component.html

```diff
@@ -28,28 +28,61 @@
     >
   </div>
 </div>
+<ng-container
+  *ngIf="!_config.features?.productConfiguratorHideMySelectionsFilterOptions"
+>
+  <div class="cx-overview-filter-option">
+    <div class="form-check">
+      <input
+        id="cx-configurator-overview-filter-option-mySelections"
+        type="checkbox"
+        class="form-check-input"
+        [formControl]="mySelectionsFilter"
+        (change)="onFilter(config)"
+        name="config-overview-my-selection-filter"
+        [attr.aria-label]="
+          'configurator.a11y.filterOverviewByMySelections' | cxTranslate
+        "
+      />
+      <label
+        class="form-check-label"
+        for="cx-configurator-overview-filter-option-mySelections"
+      >
+        {{ 'configurator.overviewFilter.mySelections' | cxTranslate }}</label
+      >
+    </div>
+  </div>
+</ng-container>
 
-<div class="cx-overview-filter-option">
-  <div class="form-check">
-    <input
-      id="cx-configurator-overview-filter-option-mySelections"
-      type="checkbox"
-      class="form-check-input"
-      [formControl]="mySelectionsFilter"
-      (change)="onFilter(config)"
-      name="config-overview-my-selection-filter"
-      [attr.aria-label]="
-        'configurator.a11y.filterOverviewByMySelections' | cxTranslate
-      "
-    />
-    <label
-      class="form-check-label"
-      for="cx-configurator-overview-filter-option-mySelections"
-    >
-      {{ 'configurator.overviewFilter.mySelections' | cxTranslate }}</label
-    >
+<ng-container
+  *ngIf="_config.features?.productConfiguratorHideMySelectionsFilterOptions"
+>
+  <div
+    *ngIf="!(isDisplayOnlyVariant() | async)"
+    class="cx-overview-filter-option"
```

Comment:
We should also remove the 'Price relevant options' filter then, shouldn't we? 
Because we cannot tell this for a variant either